src/main/scala/io/sdkman/changelogs/java/MandrelMigrations.scala

```diff
@@ -0,0 +1,31 @@
+package io.sdkman.changelogs.java
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import io.sdkman.changelogs.{
+  Mandrel,
+  Linux64,
+  Version
+}
+
+@ChangeLog(order = "035")
+class MandrelVmMigrations {
```

Comment:
just notice that this is not the filename. can you fixe it?

Reply:
Yep, fixed